ci/envs/37-minimal.yaml

```diff
@@ -1,18 +0,0 @@
-name: test
-channels:
-  - conda-forge
-dependencies:
-  - python=3.7
-  - geopandas=0.8
-  - libpysal=4.2.2
-  - networkx=2.3
-  - tqdm=4.27.0
```

Comment:
The point of `minimal` is to pin the oldest supported versions of mandatory dependencies. It is likely that we should bump those as well :D, following NEP29 but I'd like to have one env with these to ensure we actually support what we claim we do. Removing it is the reason why the coverage dropped.

Reply:
>  It is likely that we should bump those as well :D, following NEP29

Actually not, this seems to be okay. If you can just move those pins to 38-minimal, we can merge.